build.gradle

```diff
@@ -11,7 +10,6 @@ plugins {
 
 allprojects {
     apply plugin: 'java'
-    apply plugin: 'jacoco'
     apply plugin: 'com.jfrog.artifactory'
     apply plugin: 'maven-publish'
     apply plugin: 'signing'
```

Comment:
A few lines below the `gradle.projectsEvaluated {` may be removed (or added to cloud-native.java-conventions)
It adds the following lines output:
```
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```